packages/smooth_app/lib/pages/product/edit_product_page.dart

```diff
@@ -329,6 +355,19 @@ class _EditProductPageState extends State<EditProductPage> {
       },
     );
   }
+
+  void _onScrollChanged() {
+    setState(() {
```

Comment:
I don't really know about performances in flutter but maybe calling setState only when we get to the threshold could be more performant

Reply:
You're totally right, I change this